csharp-scenarios/AutoML.Net/Sonar-Forecasting.ipynb

```diff
@@ -0,0 +1,507 @@
+{
+  "cells": [
```

Comment:
@LittleLittleCloud Can you put this under - 

machine-learning/

and the dataset under 

machine-learning/data